### PR TITLE
Use the dmarc-import Elasticsearch database in the COOL DNS account

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request makes the necessary changes to allow the BOD Docker instance to use the dmarc-import Elasticsearch database in the COOL DNS account.

## 💭 Motivation and Context

The dmarc-import Elasticsearch database moved to the COOL DNS account, so the BOD Docker instance must be able to query that database in order to generate the Trustworthy Email reports.

## 🧪 Testing

I applied these changes to production and generated this week's Trustworthy Email reports.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: terraform/scripts/fetch_production_tfvars.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 TERRAFORM_TFVARS_S3_BUCKET="ncats-terraform-production-tfvars"
 TERRAFORM_DIR="terraform"
-TFVARS_FILE="production.tfvars"
+TFVARS_FILE="prod-a.tfvars"
```
Comment (dav3r) [inline terraform/scripts/fetch_production_tfvars.sh:9]: Thanks for finally changing this! 👍 I've had it changed locally forever.

**Review by dav3r — APPROVED** [review]
Very nice! 🌩️

**Review by mcdonnnj — APPROVED** [review]
Strong work cranking this out 💪 and I second @dav3r on fixing the name of the tfvars file to retrieve.

**Review by felddy — APPROVED** [review]
You've brought the Funk in a good way.  🎷

**Pull request merged**